manifest.json

```diff
@@ -6,7 +6,7 @@
         "en": "FreshRSS is a selfhostable RSS reader",
         "fr": "FreshRSS est un agrégateur de flux RSS à auto-héberger"
     },
-    "version": "1.14.2~ynh1",
+    "version": "1.14.3",
```

Comment:
please add the ~ynh1 version, so if there is another version of the packet we can increment it